FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -19,13 +19,33 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
- * http://www.FreeRTOS.org
- * http://aws.amazon.com/freertos
+ * https://www.FreeRTOS.org
+ * https://github.com/FreeRTOS
  *
+ * 1 tab == 4 spaces!
+ */
+
+/*
+ * Demo for showing how to use the Device Defender library's API. This version
```

Comment:
```suggestion
 * Demo for showing how to use the Device Defender library's APIs. This version
```

Reply:
Is this demo specific to any version of the defender library? If so please try to be specific about the version. Otherwise I think it is ok to not mention about the version here.